wrappers/swift/PrivacyGuard.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper to `ODWPrivacyGuard` representing Privacy Guard Hook.
+public class PrivacyGuard {
+    private var odwPrivacyGuard: ODWPrivacyGuard
+
+    /// Constructs object with `ODWPrivacyGuard` object which is to be wrapped around.
+    public init(privacyGuard: ODWPrivacyGuard) {
```

Comment:
I am surely missing something here, but why do we need this constructor, and also the private instance variable in this class, when all the methods in the class are static. Also, the obj-c class being wrapped around also has only class methods.

Reply:
I think you are right here. Currently the constructor for this is not needed as all methods are static. removed it.